src/ngraph/runtime/gpu/gpu_compiled_function.cpp

```diff
@@ -173,6 +173,9 @@ void runtime::gpu::GPUCompiledFunction::compile()
     pass_manager.register_pass<runtime::gpu::pass::BatchNormCache>();
     pass_manager.register_pass<ngraph::pass::LikeReplacement>();
     pass_manager.register_pass<ngraph::pass::FusedOpDecomposition>();
+    // Run this pass for the second time since, some fused operators like LSTMCell may use
+    // other fused operators inside.
+    pass_manager.register_pass<ngraph::pass::FusedOpDecomposition>();
```

Comment:
I would change `FusedOpDecomposition` to run repeatedly until there are no more unsupported `FusedOps` in the graph but that can happen in a separate PR.

Reply:
Yeah, I've plans for this, but in a separate PR.